packages/hooks/src/__tests__/useStatesList.spec.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from 'react';
+import { renderHook } from '@testing-library/react-hooks';
+import useStatesList from '../useStatesList';
+import { APIProvider, useFetch } from '@deriv/api';
+import { StoreProvider, mockStore } from '@deriv/stores';
```

Comment:
```suggestion
import * as React from 'react';
import { renderHook } from '@testing-library/react-hooks';
import { APIProvider, useFetch } from '@deriv/api';
import { StoreProvider, mockStore } from '@deriv/stores';
import useStatesList from '../useStatesList';
```

Reply:
I feel we need to copy the config made my bots to accounts. The ordering of imports should not be our concern

Reply:
should we create a card for it?